plutus-benchmark/marlowe/exe/PlutusBenchmark/Marlowe/Semantics.hs

```diff
@@ -61,7 +61,7 @@ validatorHash = marloweValidatorHash
 
 -- | The benchmark cases for the Marlowe semantics validator.
 benchmarks :: IO (Either String [Benchmark])
-benchmarks = second (rescript <$>) <$> readBenchmarks "semantics"
+benchmarks = second (rescript <$>) <$> readBenchmarks "marlowe/exe/scripts/semantics"
```

Comment:
I was wondering why it finds these files OK no matter where you are, but I see that `readBenchmarks` calls `getDataDir` which returns the root of `plutus-benchmarks`.